test/other/metadce/test_metadce_files_wasmfs.size

```diff
@@ -1 +1 @@
-51616
+60220
```

Comment:
This seems like a pretty big bump that might be worth investigating.   Is print being pulled in where it wasn't before for some reason?

There are a few useful linking command line tricks for tracking down why symbols get included.  For example `--trace-symbol` and maybe `-Wl,-Map=`.

Reply:
Yeah this also turned out to be the `__libcpp_verbose_abort` issue. We thought adding '-D_LIBCPP_AVAILABILITY_HAS_NO_VERBOSE_ABORT` in libcxx's `system_libs.py` would solve the issue, but those routines are in the headers. So [these routines](https://github.com/llvm/llvm-project/blob/beb121f6322be466384ffaa68b913f58862ed14c/libcxx/include/stdexcept#L227-L305) can be called whenever a user gives errorneous arguments to `string` or `vector` allocations, so basically any user code can pull these `__libcpp_verbose_abort` in. In this case libwasmfs code pulled in the `printf` family. We can add `-D_LIBCPP_AVAILABILITY_HAS_NO_VERBOSE_ABORT` to libwasmfs's `cflags` in `system_libs.py` but that's just a band-aid for libwasmfs because we can't do that for all user code.

I guess we should just define it in https://github.com/emscripten-core/emscripten/blob/main/system/lib/libcxx/include/__verbose_abort. WDYT?

Reply:
Ping 😀

Reply:
How about defining it `system/lib/libcxx/include/__config_site`

Reply:
Done